### DISCUSSION TITLE
Info tecniche sul database

### DISCUSSION THREAD

**Discussion by maelaborazioni** [globaleaks/globaleaks-whistleblowing-software#3725] [Q&A] [answered]
Buongiorno,
da ciò che leggo sulla pagina con la documentazione vengono critptati dati, allegati, messaggi e metadati scambiati tra segnalante e gestore tramite il protocollo di criptazione di GlobaLeaks. Quindi se ho capito bene la cifratura è a livello dell'applicativo.
È prevista la cifratura anche del database? Con quale algoritmo di cifratura?
È prevista la separazione di dati in distinti database? 
I dati identificativi sono conservati separatamente dagli altri?
È prevista pseudonimizzazione o anonimizzazione di alcuni dati?

**Comment by evilaliv3** [comment, marked as answer]
Buonasera, grazie per la domanda.

Il software memorizza le informazioni nella cartella /var/globaleaks (file e database) che vengono cifrati con protocollo applicativo: https://docs.globaleaks.org/en/main/security/EncryptionProtocol.html
Ogni dato e metadato di segnalazione viene dunque cifrato.

I sistema effettua anonimizzazione e pseudoanonimizzazione dove possibile in ogni forma; ad esempio metadati di connessione non vengono registrati i dati identificativi vengono mantenuti in tabelle logicamente distinte e a seconda delle configurazioni resi fruibili ai soli utenti autorizzati.

**Reply by maelaborazioni** [reply]
Grazie mille per la esaustiva e pronta risposta!
